src/ontodia/data/rdf/rdfCacheableStore.ts

```diff
@@ -0,0 +1,255 @@
+import { RDFStore, RDFGraph, createStore, createGraph, Node, Literal, NamedNode, Triple } from 'rdf-ext';
+import { Dictionary } from '../model';
+import { RDFCompositeParser } from './rdfCompositeParser';
+
+import { uniqueId } from 'lodash';
+
+const DEFAULT_STORAGE_TYPE = 'text/turtle';
+const DEFAULT_STORAGE_URI = 'ontodiaLocalData';
```

Comment:
`ontodiaLocalData` is still technicallly not a URI. For example, you could change it to `ontodia:localData`